tests/FVTTests/fileParameterView/testFileParameterView.ts

```diff
@@ -70,17 +66,25 @@ describe('JES explorer spool file in url query (explorer-jes/#/viewer)', functio
         await loadUrlWithSearchFilters(driver, filters);
         const jobObjs :ParsedJobText[] = await waitForAndExtractParsedJobs(driver);
         expect(jobObjs && jobObjs.length > 0).to.be.true;
+
         testFilters = {
             jobName: jobObjs[0].prefix,
             jobId: jobObjs[0].jobId,
-            fileId: 3,
-        };
+            fileId: 108,
+        }
+
+        // load driver with specified filter
         await loadUrlWithViewerFilters(driver, testFilters);
+ 
+    });
 
-        // wait for content to load
-        const viewer = await driver.findElement(By.css('#embeddedEditor > div > div > .textviewContent'));
-        const text = await viewer.getText();
-        expect(text).to.have.lengthOf.greaterThan(1);
+    it('Should handle rendering file contents in Orion editor', async () => {
```

Comment:
Mocha executes tests sequentially based on the file order, I think this test belongs at the bottom of this file so that the the tests increase in complexity and follow the page order

Reply:
Got it. I will move this test to the bottom of the file.